modules/rosa-sdpolicy-platform.adoc

```diff
@@ -68,6 +68,24 @@ ifndef::openshift-rosa-hcp[]
 endif::openshift-rosa-hcp[]
 clusters at this time. However, custom labels are supported when creating new machine pools.
 
+[id="rosa-sdpolicy-node-lifecycle_{context}"]
+== Node lifecycle
+
+Worker node longevity is not guaranteed and may be replaced at any time as part of the normal operation and management of OpenShift.
+
+Several events affect the longevity of a {product-title} worker node:
+
+* Machine health checks are deployed and configured to ensure that a worker node with a `NotReady` status is replaced to ensure smooth operation of the cluster.
+* AWS EC2 instance longevity may occur at any time when AWS detects irreparable failure of the underlying hardware that hosts the instance.
```

Comment:
"AWS EC2 instance longevity" seems like an unusual noun here - if longevity is a specific AWS field it likely needs to be marked up with backticks.
"AWS EC2 instances may be terminated when..."?